superagi/controllers/models_controller.py

```diff
@@ -121,7 +121,7 @@ def get_knowledge_list(page: int = 0, organisation=Depends(get_user_organisation
 
 @router.get("/marketplace/list/{page}", status_code=200)
 def get_marketplace_knowledge_list(page: int = 0):
-    organisation_id = 2
+    organisation_id = 13
     page_size = 16
```

Comment:
Don't hardcode it see how it is done in tools and knowledge we have it in config.yaml we can use from there.